vendor/code.cloudfoundry.org/cli/cf/configuration/coreconfig/config_data.go

```diff
@@ -46,7 +46,7 @@ type Data struct {
 func NewData() *Data {
 	data := new(Data)
 
-	data.UAAOAuthClient = "cloudfoundry"
+	data.UAAOAuthClient = "cf"
```

Comment:
@janosbinder any idea of how this change impacts terraform-cloudfoundry-provider users ? 

The default in cf-deployment is still to use a "cf" uaa client, see https://github.com/cloudfoundry/cf-deployment/blob/a456c2805b7623a9c241294fc09d2a65479b6ea5/cf-deployment.yml#L531-L538

I don't understand how this could have worked before ? Maybe it's dead code.

Reply:
I think it was renamed by a mistake in #44 . See the changes here https://github.com/mevansam/terraform-provider-cf/blob/0.9.9/vendor/code.cloudfoundry.org/cli/cf/configuration/coreconfig/config_data.go by selecting "blame".

In the previous commit it used to be "cf" as well, see here https://github.com/cloudfoundry/cli/blob/3379b668c1061a67f3762366e4cda6388559f927/cf/configuration/coreconfig/config_data.go#L49 The `govendor sync` restored the correct contents.

The reason, why we have not faced any issue, as we did not use this part of the code.

In summary, in the `vendor` folder I would just only change the `vendor.json` in the future. Let me know, if the team has anything against this idea.